basics-assignment-2-start 2/src/app/app.component.html

```diff
@@ -0,0 +1,12 @@
+<!-- <ol>
+  <li>Add a Input field which updates a property ('username') via Two-Way-Binding</li>
+  <li>Output the username property via String Interpolation (in a paragraph below the input)</li>
+  <li>Add a button which may only be clicked if the username is NOT an empty string</li>
+  <li>Upon clicking the button, the username should be reset to an empty string</li>
+</ol> -->
+
+<h1>username</h1>
+<input [(ngModel)]="username" placeholder="Enter your username" class="form-control"/>
+<p>{{username}}</p>
+<div *ngIf="username.length == 0, else elseBlock"><button (click)="reset()" [disabled]="true" class="btn btn-primary">Reset User</button></div>
+<ng-template #elseBlock><button (click)="reset()" class="btn btn-primary">Reset User</button></ng-template>
```

Comment:
Don't render two components. this should be done with one and only one button element.